### DISCUSSION TITLE
JVM support

### DISCUSSION THREAD

**Discussion by jQrgen** [touchlab/CrashKiOS#90] [Feature request]
Hey ho

Is there any plan for adding JVM target support?

**Comment by wellingtoncosta** [comment]
Hi, we're planning to add no-op support for JVM / JS. I just created this discussion: https://github.com/touchlab/CrashKiOS/discussions/101

**Comment by wellingtoncosta** [comment]
As describred in #101, for now we added no-op support for JVM and JS. The main reason is that Crashlytics supports Android and iOS only, altho Bugsnag does support JVM and JS.
